plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -182,10 +182,22 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
       return
     }
     // move native map to new position
-    mapCameraManagerDelegate.setCamera(cameraOptions)
-    // notify listeners with actual values
-    notifyListeners(mapCameraManagerDelegate.cameraState)
-    lastCameraOptions = cameraOptions
+    try {
+      mapCameraManagerDelegate.setCamera(cameraOptions)
+      // notify listeners with actual values
+      notifyListeners(mapCameraManagerDelegate.cameraState)
+      lastCameraOptions = cameraOptions
+    } catch (e: Exception) {
+      Logger.e(
+        TAG,
+        "Exception while setting camera options : ${e.message} CameraOptions = $cameraOptions"
+      )
+    } catch (error: Error) {
```

Comment:
can we add a unit test making sure that when exception happens we do not update  `lastCameraOptions` and do not notify listeners?